### PR TITLE
Add popup menu

### PR DESCRIPTION
**Pull request by Grandi0z**

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @Grandi0z,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

❌You have submitted a project, that has no contribution from the pair programming partner. Kindly check the [Work and Submission Mode](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#work-and-submission-mode) ❌

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style/styles.css [hunk 1]
```diff
@@ -594,6 +603,95 @@ textarea {
...
+    top: 0;
+    right: 0;
+    width: 100%;
+    max-height: 100vh;
+    z-index: 99;
+    margin-top: 0;
+    background-color: #fff;
+    overflow-y: auto;
+    box-sizing: border-box;
+  }
```
Comment (OyePriscilla) [inline style/styles.css]: - [ ] Please kindly add margin to your popup window for both desktop and mobile version. Part of our duty as developers is our capability to reproduce any design given to us expected. 👍 

## Figma design for desktop version

q2

## Figma design for mobile version
q3

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Grandi0z 😃, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 

## Required Changes ♻️
__ kindly check the inline comments below__

## Optional suggestions
[optional] Please kindly increase the size of your `nav items   id="deskNav"`for the desktop version.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @Grandi0z 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**